app/src/views/Detalhamento.vue

```diff
@@ -102,18 +209,140 @@ export default {
       if (this.order.Prioridade_idPrioridade === 4) return 'very-high-priority';
     },
+    async getManutentor() {
+      try {
+        const response = await this.$http.post('detalhamento/getgeraluser', getLocalStorageToken(), this.valuesInput);
+
+        if(response.result.length === undefined)
+          this.manutentores.push(response.result);
+        else this.manutentores = [ ...response.result ];
+      } catch (err) {
+        this.$swal({
+          type: 'warning',
+          title: getErrors(err),
+          confirmButtonColor: '#F34336',
+        });
+      }
+    },
+    // manutentorInOrdem
+    async getManutentoresInOrdem() {
+      try {
+        const response = await this.$http.post('detalhamento', getLocalStorageToken(), this.valuesInput);
+        if(response.result.length === undefined)
+          this.manutentorInOrdem.push(response.result);
+        else this.manutentorInOrdem = [ ...response.result ];
+      } catch (err) {
+        return this.$swal({
+          type:'warning',
+          title: getErrors(err),
+          confirmButtonColor: '#F34336',
+        });
+      }
+    },
+    validaAddManutentor(User) {
+      const result = this.manutentorInOrdem.find(element => element.idUsuario === User.idUsuario);
+      return result;
+    },
+    async addManutentor(index, row){
+      try {
+        const validManutentorAdd = this.validaAddManutentor(row);
+
+        if(validManutentorAdd === undefined){
+          this.dialogVisible = false;
+          this.visibleMessage = false;
+          this.valuesInput.excluded = 0;
+          this.valuesInput.idUsuario = row.idUsuario;
+
+          const response = await this.$http.post('detalhamento/register', getLocalStorageToken(), this.valuesInput);
+
+          this.$swal({
+            type: 'success',
+            title:`${response.result}`,
+            confirmButtonColor: '#F34336',
+          }),
+
+          this.getManutentoresInOrdem();
+
+          this.$http.setActivity(
+            'registerUser',
+              {
+                ...this.$store.state.user,
+                date: this.$moment().format('DD-MM-YYYY HH-mm'),
+                descricao: `${this.$store.state.user.nome} 
+                registerUser o usuário ${row.nome} para ajudar na ordem serviço`,
+              },
+            getLocalStorageToken(),
+          );
+        }else {
+          this.visibleMessage = true;
+        }
+      } catch (err) {
+        return this.$swal({
+          type: 'warning',
+          title: getErrors(err),
+          confirmButtonColor: '#F34336',
+        });
+      }
+    },
+    async RemoveManutentor(index, row){
+      try {
+        this.valuesInput.idUsuario = row.idUsuario;
+        this.valuesInput.excluded = 1;
+        this.dialogVisible = false;
+        const response = await this.$http.update('detalhamento', getLocalStorageToken(), this.valuesInput, this.valuesInput.idOrdemServico);
+        if(response.status === 200) return this.$swal({
```

Comment:
Esse item aqui, se status for 200 ele vai retornar a mesangem e não irá executar o bloco de código a baixo, isso está certo? se remover ele vai exibir a msg de que foi removido mas no final n vai remover?